### ISSUE TITLE
Fix HTML containing `<template>` element

### ISSUE THREAD

**Issue by SaekiTominaga** [markuplint/markuplint#606]
- Markuplint version: `3.0.0-rc.1`
- Parser lang: HTML
- Node.js version: `18.12.0`
- OS: Windows

## Describe the bug

Errors are reported even in valid HTML when `<template>` element is included. 
Note, no problem with version 2 (2.11.3).

## Code Example

```html
<!-- error: ロール「rowgroup」にはロール「row」が必要です (wai-aria) -->
<table>
  <caption>
    text
  </caption>
  <tbody>
    <template>
      <tr>
        <td></td>
      </tr>
    </template>
  </tbody>
</table>

<!-- error: ロール「list」にはロール「listitem」が必要です (wai-aria) -->
<ul>
  <template>
    <li></li>
  </template>
</ul>

<!-- error: 1つ以上の要素が必要です。(「dt」が必要です) (permitted-contents) -->
<dl>
  <template>
    <dt></dt>
    <dd></dd>
  </template>
</dl>
```

**Comment by YusukeHirao** [comment]
@SaekiTominaga Thank you for creating this issue.

In the WAI-ARIA's problems, you are right, I'll fix it while checking that.

But, I think it is correct about the warning that a `dl` element only includes a `template` element.

Because HTML Standard says:

> Zero or more groups each consisting of one or more [dt](https://html.spec.whatwg.org/multipage/grouping-content.html#the-dt-element) elements followed by one or more [dd](https://html.spec.whatwg.org/multipage/grouping-content.html#the-dd-element) elements, optionally intermixed with [script-supporting elements](https://html.spec.whatwg.org/multipage/dom.html#script-supporting-elements-2).

https://github.com/markuplint/markuplint/blob/5faf276d05849b4b239b0b436158fc9f43eb12ca/packages/%40markuplint/html-spec/src/spec.dl.json#L11-L27

**Comment by YusukeHirao** [comment]
@SaekiTominaga 

I had implemented it doesn't warn if the element has `aria-busy` but, I added the message about **it needing `aria-busy`** if it doesn't.
(I will add the details to the documents.)

**Comment by YusukeHirao** [comment]
@SaekiTominaga 

I publish `v3.0.0-rc.3` just now.

**Comment by SaekiTominaga** [comment]
Thanks for the update. I have confirmed the message change in `v3.0.0-rc.3`.

> But, I think it is correct about the warning that a `dl` element only includes a `template` element.

Sorry, please forget about this.
